README.md

```diff
@@ -140,6 +140,12 @@ The following values are returned in `results.csv`:
 * `DMARC Policy` - An adjudication, based on any policies found in
   `DMARC Results` and `DMARC Results on Base Domain`, of the relevant
   DMARC policy that applies.
+* `DMARC Policy Percentage` - The percentage of mail that should be subjected to the `DMARC Policy`
```

Comment:
Maybe we shoud add an error if ‘pct‘ is less than 100? To me, having a lesser percentage defeats the entire point of a gradual tightening of the policy, because it means it's not being applied to an arbitrary amount of email.

E.g. usaid.gov currently has a 0 percentage set :/

Reply:
I agree, but I'd like to hear what @h-m-f-t thinks.

Reply:
For what it's worth. I changed this condition from a warning to an error in my own checker too, to avoid misleading people into thinking their setup is fine when it is not.